docs/build.py

```diff
@@ -213,8 +213,8 @@ def main() -> int:
         "-t",
         "--timeout",
         dest="timeout",
-        help="Timeout (in seconds) for sphinx-build (default=180)",
-        default=180,
+        help="Timeout (in seconds) for sphinx-build (default=360)",
+        default=360,
```

Comment:
Did you meant to change this?

Reply:
Yes, this was an intended change that I asked Dan about. When I ran build.py, it would timeout every time. He said it was okay to change this so it could be a topic of discussion

Reply:
The workflow timeout is 600 seconds: https://github.com/IDAES/idaes-pse/blob/7eac46139444f0cad681ede7154151123726f13b/.github/workflows/core.yml#L178. If the current build timeout value is too low, we could just bump it up to 600.